xarray/plot/dataset_plot.py

```diff
@@ -37,119 +37,119 @@
 
 def _dsplot(plotfunc):
     commondoc = """
-    Parameters
```

Comment:
Took me an annoying long time to figure out that decreasing the indention on these solved the following sphinx/autodoc warnings: 
```
/Users/scotthenderson/GitHub/xarray/xarray/xarray/plot/accessor.py:docstring of xarray.plot.accessor.DatasetPlotAccessor.quiver:7: CRITICAL: Unexpected section title.

Parameters
----------
xarray/plot/accessor.py:docstring of xarray.plot.accessor.DatasetPlotAccessor.quiver:11: ERROR: Unexpected indentation.
xarray/plot/accessor.py:docstring of xarray.plot.accessor.DatasetPlotAccessor.quiver:12: WARNING: Block quote ends without a blank line; unexpected unindent.
xarray/plot/accessor.py:docstring of xarray.plot.accessor.DatasetPlotAccessor.quiver:117: WARNING: Inline strong start-string without end-string.
```